### ISSUE TITLE
starting npm from phomium

### ISSUE THREAD

**Issue by dave21458** [dave21458/Phomium#6]
Nothing is easy.......
After messing with what should have been simple all day trying to get npm to start via phomium, like an idiot i discovered that npm and node and whatever must be in the same directory as phomium. It will not start no matter what else I do... Ok fine no big...
You will love this part....
in phomium I'm starting npm... which starts node then dies, which phomium thinks it failed to start... so when phomium closes it can not close node because it did'nt start it.
My question is can npm stay alive until it is closed by phomium then it would close node?
being a windows script I would think it could..or not

**Comment by joshuaebowling** [comment]
Yes, my answer would be a lib called forever. You may have to trick it to run locally, is typically a global install. It runs until you stop it. 

Forever js, its on npm wesbite

**Comment by joshuaebowling** [comment]
Also maybe a powershell option, after using Linux terminal for so long, cmd is sooooo weak lol
